src/lib/components/molecules/ProjectCard.svelte

```diff
@@ -3,13 +3,33 @@
 	import Tag from '$lib/components/atoms/Tag.svelte';
 	import Image from '$lib/components/atoms/Image.svelte';
 	import GitHubIcon from '$lib/icons/socials/github.svelte';
+	import Star from '$lib/icons/star.svelte';
+	import { onMount } from 'svelte';
 
 	export let title: string;
 	export let coverImage: string | undefined = undefined;
 	export let tags: string[] | undefined = undefined;
-	export let repo: string | undefined = undefined;
+	export let repoURL: string | undefined = undefined;
+	export let repoName: string | undefined = undefined;
 
 	export let showImage = true;
+	export let stars: number | null = null;
+
+	async function getStars() {
```

Comment:
Hi @grmbyrn I would add an argument to the function `repoName` instead of using the global variable. I think it's better if we explicitly define function inputs.